src/app/resources/resources-view.component.ts

```diff
@@ -6,6 +6,7 @@ import { ActivatedRoute, ParamMap } from '@angular/router';
 import { DomSanitizer } from '@angular/platform-browser';
 import { Router } from '@angular/router';
 import { environment } from '../../environments/environment';
+import { Jsonp, Response } from '@angular/http';

```

Comment:
angular new version uses import { HttpClientJsonpModule } from '@angular/common/http';

Reply:
Also the `Response` class is not used in the component.  Please remove it.

Reply:
there is separate issue for HttpClientJsonpModule https://github.com/ole-vi/planet/issues/245